index.html

```diff
@@ -0,0 +1,163 @@
+                    <li><a href="#myPage"></a>My Page</li>
+                    <li><a href="#logout"></a>Logout</li>                   
+                </ul>
+            </div>
+        </div>
+        <div class="header-nav">
+            <div class="navbar">
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+                <div class="logo">
+                    <a href="index.html"><img src="./assets/images/logo.PNG" alt="merctech mentorship"></a>
+                </div>
+                <nav class="nav-menu">
+                    <ul class="menu-links">
+                      <li>
+                        <a href="about.html" class="a-links">About</a>
+                      </li>
+                      <li>
+                        <a href="#program" class="a-links">Program</a>
+                      </li>
+                      <li>
+                        <a href="#join"class="a-links">Join</a>
+                      </li>
+                      <li>
+                        <a href="#sponsor"class="a-links">Sponsor</a>
+                      </li>
+                      <li>
+                        <a href="#news"class="a-links">News</a>
+                      </li>
+                      <li class="campaign">
+                        <a href="#campaign"class="a-links">campaign</a>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+    </header>
+    <section class="hero-container">
+        <div class="container-herooo">
+            <div class="hero">
+                <div class="hero-image">
+                    <img src="./assets/images/theme11.png" class="theme" alt="theme">
+                </div>
+                
+                <div class="sub-theme">
+                    <p>We are a nonprofit organization whose goal is to build a pipeline to STEM fields (Science, Technology, Engineering, Math) for underprivileged and underrepresented groups to encourage and promote diversity.</p>
+                </div>
+                <div class="data-venue">
+                    <p class="time">2023.10.15(THU) ~ 16(FRI)</p>
+                    <p class="venue">@ National Museum of Kenya, Nairobi Centermore</p>
+                </div>
+            </div>
+        </div>
+        
+    </section>
+    <main>
+        <div class="program">
+        <div class="program-content">
+            <h3 class="title">Main Program</h3>
+            <p class="title-underline"></p>
+        </div>
+        <div class="program-details">
+            <div class="program-card">
+                <img src="./assets/icons/program_icon_01 1.png" class="program-icon" alt="image1"/>
+                <h3 class="program-text">Networking</h3>
+                <p class="program-cont">Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
+            </div>
```

Comment:
Here `Networking` (or whatever text is here) should be centered, and the `lorem ipsum` below should be centered as well, this would balance the design better. 💯 Also make use of padding/margins to get the design to breathe better, the `lorem ipsum` shouldn't be touching the borders. ❌ 

![Screenshot 2023-03-18 at 14 31 10](https://user-images.githubusercontent.com/45739963/226128767-86ee71df-085a-4aaa-ad68-62ec88365342.png)